### ISSUE TITLE
[NBug] Unable to parse object ID "".

### ISSUE THREAD

**Issue by KeidyU** [gitextensions/gitextensions#8270]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Error Details
```
System.FormatException: Unable to parse object ID "".
   at GitUIPluginInterfaces.ObjectId.Parse(String s)
   at GitCommands.GitModule.GetCurrentCheckout()
   at GitUI.CommandsDialogs.FormCheckoutBranch.OkClick()
   at GitUI.GitUICommands.<>c__DisplayClass71_0.<StartCheckoutBranch>b__0()
   at GitUI.GitUICommands.DoActionOnRepo(IWin32Window owner, Boolean requiresValidWorkingDir, Boolean changesRepo, EventHandler`1 preEvent, EventHandler`1 postEvent, Func`1 action)
   at GitUI.GitUICommands.StartCheckoutBranch(IWin32Window owner, String branch, Boolean remote, IReadOnlyList`1 containRevisions)
   at GitUI.CommandsDialogs.FormBrowse.<>c__DisplayClass164_0.<CurrentBranchDropDownOpening>b__3(Object <p0>, EventArgs <p1>)
   at System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.ToolStripDropDown.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

## Environment
- Git Extensions 3.1.1.6049
- Build 2f872105b4eb42421d9fdafd68e90a9e27d20d96
- Git 2.23.0.windows.1
- Microsoft Windows NT 10.0.18362.0
- .NET Framework 4.8.4180.0
- DPI 120dpi (125% scaling)

**Comment by pmiossec** [comment]
Should be solved. Please update to last version.

**Comment by gerhardol** [comment]
Duplicate of #6512
Corrected in #6513, included in 3.2
